packages/blade-old/src/_helpers/isValidElement.js

```diff
@@ -0,0 +1,8 @@
+import { isValidElementType } from 'react-is';
+
+const isValidElement = (element) => {
```

Comment:
why not name this as `isValidElementType`?

Reply:
isValidElement is smaller and doesn't clash with react-is's function name. 
naming is hard :p let's keep it this way?